### PR TITLE
support geo encoder

### PR DESCRIPTION
**Pull request by jievince**

### CODE REVIEW

File: client/src/main/java/com/vesoft/nebula/encoder/RowWriterImpl.java [hunk 1]
```diff
@@ -626,6 +636,29 @@ public void write(int index, DateTime v) {
...
+                throw new RuntimeException("Incorrect geo shape, expect "
+                        + field.geoShape() + ", got "
+                        + v.getSetField());
+            }
+        } else {
+            throw new RuntimeException("Value: " + v + "'s type is unexpected");
+        }
+        org.locationtech.jts.geom.Geometry jtsGeom = convertGeographyToJTSGeometry(v);
+        byte[] wkb = new org.locationtech.jts.io
+                         .WKBWriter(2, ByteOrderValues.LITTLE_ENDIAN)
```
Comment (Nicole00) [inline client/src/main/java/com/vesoft/nebula/encoder/RowWriterImpl.java:657]: Use byteOrder defined in construct function, which is the native order.

File: client/src/test/java/com/vesoft/nebula/encoder/TestEncoder.java [hunk 2]
```diff
@@ -62,8 +76,39 @@
...
+        shell.add(new Coordinate(-108.7, 35.0));
+        List<Coordinate> hole = new ArrayList<>();
+        hole.add(new Coordinate(-100.1, 41.4));
+        hole.add(new Coordinate(-102.9, 37.6));
+        hole.add(new Coordinate(-96.8, 37.5));
+        hole.add(new Coordinate(-100.1, 41.4));
+        List<List<Coordinate>> rings = new ArrayList<List<Coordinate>>();
+        rings.add(shell);
+        rings.add(hole);
+        geogPolygonVal.setGgVal(Geography.pgVal(new Polygon(rings)));
```
Comment (Nicole00) [inline client/src/test/java/com/vesoft/nebula/encoder/TestEncoder.java:108]: please add test for null geograph value.

File: client/src/main/java/com/vesoft/nebula/encoder/RowWriterImpl.java (hunk 1 above)
Comment (jievince) [reply client/src/main/java/com/vesoft/nebula/encoder/RowWriterImpl.java:657]: What is native order? The default byte order of jts's WKBWriter is `BIG_ENDIAN`, I change it to `little_endian` is because nebula-graph's WKBWriter use the machine's byte order as a default, which is of `little_endian`.

File: client/src/test/java/com/vesoft/nebula/encoder/TestEncoder.java (hunk 2 above)
Comment (jievince) [reply client/src/test/java/com/vesoft/nebula/encoder/TestEncoder.java:108]: It's not a need, because null geo value is no different from other types. And TestEncoder.java also just test null value of int64

File: client/src/main/java/com/vesoft/nebula/encoder/RowWriterImpl.java (hunk 1 above)
Comment (Nicole00) [reply client/src/main/java/com/vesoft/nebula/encoder/RowWriterImpl.java:657]: native order is the machine's byte order. 

you use the little_endian as the default order, what if user's machine's byte order is big_endian?

File: client/src/test/java/com/vesoft/nebula/encoder/TestEncoder.java (hunk 2 above)
Comment (Nicole00) [reply client/src/test/java/com/vesoft/nebula/encoder/TestEncoder.java:108]: It's necessary, it happens `out of bound error` before for null string value int the properties. So you should add test like `there are three geography properties but one is null` to avoid that problem.

File: client/src/main/java/com/vesoft/nebula/encoder/RowWriterImpl.java (hunk 1 above)
Comment (jievince) [reply client/src/main/java/com/vesoft/nebula/encoder/RowWriterImpl.java]: OK, Thanks for your help

File: client/src/test/java/com/vesoft/nebula/encoder/TestEncoder.java (hunk 2 above)
Comment (jievince) [reply client/src/test/java/com/vesoft/nebula/encoder/TestEncoder.java:108]: OK

**Review by Nicole00 — APPROVED** [review]
LGTM
